yoyodyne/train.py

```diff
@@ -27,13 +27,9 @@ def _get_logger(experiment: str, model_dir: str, log_wandb: bool) -> List:
     """
     trainer_logger = [loggers.CSVLogger(model_dir, name=experiment)]
     if log_wandb:
-        trainer_logger.append(
-            loggers.WandbLogger(project=experiment, log_model="all")
-        )
-        # Tells PTL to log best validation acc
+        trainer_logger.append(loggers.WandbLogger(project=experiment))
+        # Tells PTL to log the best validation accuracy.
         wandb.define_metric("val_accuracy", summary="max")
-        # Logs the path to local artifacts made by PTL.
-        wandb.config.update({"local_run_dir": trainer_logger[0].log_dir})
```

Comment:
I had this here because if you are running a large sweep somewhere, and later want to e.g. get predictions from certain W&B configs, this is easy to do with a script by just getting the CSV, and selecting rows representing runs. Then we just need row["local_run_dir"] to get the checkpoint.

This is especially helpful for sweeps where several runs crashed, and you want to easily differentiate completed from crashed runs. Do you think its worth keeping? Or is it too specific to my needs?

Reply:
If you think it's useful, I'll restore it. I think I can write it slightly more clearly.

Reply:
Restored, but I'm using dict-style syntax. LMK if this looks okay...it's working fine for me locally.

Reply:
Ah I did not know we could do that---yeah it looks better!